core/api/BUCK

```diff
@@ -180,21 +180,98 @@ dev_pnpm_task_test(
 
 jest_test(
     name = "unit-tests",
-    srcs = [":src"] + [":test_src"] + glob([".env", "galoy.yaml"]),
+    srcs = [":src"] + [":test_src"] + glob(["galoy.yaml"]),
     config_file = "test/unit/jest.config.js",
-    env_file = ".env",
```

Comment:
`env_json = "//dev:api-env"`

Reply:
Context: these variables are pretty much dev environment related so nesting at the `dev/` dir level makes sense